draft-ietf-quic-transport.md

```diff
@@ -4084,23 +4088,28 @@ fragmentation on the path.
 A client MUST expand the payload of all UDP datagrams carrying Initial packets
 to at least the smallest allowed maximum packet size (1200 bytes) by adding
 PADDING frames to the Initial packet or by coalescing the Initial packet; see
-{{packet-coalesce}}.  Sending a UDP datagram of this size ensures that the
-network path from the client to the server supports a reasonable Path Maximum
-Transmission Unit (PMTU).  This also helps reduce the amplitude of amplification
-attacks caused by server responses toward an unverified client address; see
+{{packet-coalesce}}.  Similarly, a server MUST expand the payload of all UDP
+datagrams carrying ack-eliciting Initial packets.  Sending UDP datagrams of this
+size ensures that the network path supports a reasonable Path Maximum
+Transmission Unit (PMTU), in both directions.  Additionally, a client padding
+Initial packets helps reduce the amplitude of amplification attacks caused by
+server responses toward an unverified client address; see
```

Comment:
Can you spell it out fully for the server? "A server MUST similarly expand the payload of all UDP datagrams carrying ack-eliciting Initial packets to at least the smallest allowed maximum packet size (1200 bytes)."

Also, "a client padding Initial packets" --> "a client that expands Initial packets"

Reply:
Thank you for the suggestions. Done in c550cb4.